docs/process-services/process-list.component.md

```diff
@@ -43,7 +43,7 @@ Renders a list containing all the process instances matched by the parameters sp
 | Name | Type | Default value | Description |
 | -- | -- | -- | -- |
 | appId | `number` |  | The id of the app. |
-| data | [`DataTableAdapter`](../../lib/core/datatable/data/datatable-adapter.ts) |  | Data source to define the datatable. |
+| data | [`DataTableAdapter`](../../lib/core/datatable/data/datatable-adapter.ts) |  | Data source to define the datatable. **Deprecated:** in 2.4.0  |
```

Comment:
the deprecated comment should be also in top of the input parameter because the doc processor is going to change automatically this part if you don't add it even there the next time  we regenerated will not be present